pages/ProfileVolunteer.tsx

```
@@ -0,0 +1,270 @@
+                        <Text style={styles1.sectionTitle}>{t('myCalendar')}</Text>
+                        <Calendar />
+                    </View>
+
+                    <View style={styles1.statsCard}>
+                        <View style={styles1.statsContent}>
+                            <Text style={styles1.statsLabel}>
+                                {t('accomplishedMissions')}
+                            </Text>
+                            <Text style={styles1.statsNumber}>{missionsAccomplies}</Text>
+                        </View>
+                        <ImageButton onPress={true ? undefined : navToRewards} image={require('@/assets/images/reward.png')} />
```

Biome (2.3.13)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)